src/climate_assessment/cli.py

```diff
@@ -245,7 +245,15 @@
     type=bool,
     show_default=True,
 )
-
+save_csv_combined_output_option = click.option(
+    "--save-csv-combined-output",
```

Comment:
maybe call it `save_csv_combined_climate_output_option` to be a bit more descriptive (it does not include emissions data, right?) and align better with `save_raw_climate_output_option`

Reply:
It does include emissions, it's the `rawoutput` file. That's why I went for "combined", but suggestions on a more descriptive name are appreciated

Reply:
Ah yes you're right. 

Okay, rather then thinking about it too long, I think what can simply do is add a bit more description in the docs: https://github.com/iiasa/climate-assessment/issues/44
Can be done after we merge this PR.